client/src/components/FeedbackForm/FeedbackButton.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import renderComponent from 'utils/testing';
+import configureStore from 'redux-mock-store';
+import FeedbackButton from './FeedbackButton';
+import { initialTicketCreateState as ticketCreate } from '../../redux/reducers/tickets.reducers';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
I believe this is unused.

- [`testing-library/jest-dom` at "Custom matchers"](https://github.com/testing-library/jest-dom#custom-matchers) (none of these are used)

```suggestion
```